tasks/views.py

```diff
@@ -1,3 +1,13 @@
-# from django.shortcuts import render
+from django.shortcuts import render
+from tasks.models import Task
+from users.models import Role, User
 
-# Create your views here.
+
+def view_tasks(request):
+    context = {}
+    if not request.user.is_anonymous:
+        app_user = User.objects.get(user=request.user)
+        context['tasks'] = Task.filter_by_assignee(request.user.id) if app_user.role == Role.EMPLOYEE \
+            else Task.filter_by_team(app_user.team)
```

Comment:
It's more recommended to avoid using a backslash at the end of the line one reason is that it prevent from you adding a comment for that line for more details look at:
https://stackoverflow.com/questions/10660435/pythonic-way-to-create-a-long-multi-line-string/56013473#56013473
so you can use parentheses instead like:
```suggestion
        context['tasks'] = Task.filter_by_assignee(request.user.id) if app_user.role == Role.EMPLOYEE else (
        Task.filter_by_team(app_user.team)
        )
```

Reply:
Resolved.